eva/executor/load_multimedia_executor.py

```diff
@@ -93,11 +90,8 @@ def exec(self):
 
             storage_engine = StorageEngine.factory(table_obj)
             if do_create:
-                success = storage_engine.create(table_obj)
-                if not success:
-                    raise ExecutorError(
-                        f"StorageEngine {storage_engine} create call failed"
-                    )
+                storage_engine.create(table_obj)
```

Comment:
Here do still need to check the success from `create`? We probably don't want the next write if create failed.

Reply:
\cc @gaurav274

Reply:
It is fine. `create` will raise error